src/Idea.js

```diff
@@ -13,17 +13,28 @@ const style = {
 
 
 
-export const Idea = ({id,left,top,hideSourceOnDrag,children, }) => {
+export const Idea = ({id,left,top,hideSourceOnDrag,children,displayEdit }) => {
```

Comment:
`displayEdit` should be renamed to something that starts with "on" as that's the convention for indicating that it's a callback.